### ISSUE TITLE
how to install the gmml to my game?

### ISSUE THREAD

**Issue by Ceyase** [cgytrus/gmml#1]
This is a great project,but it doesn't seem to work.

**Comment by cgytrus** [comment]
yeah it's pretty difficult to compile yourself rn so you can't really do anything if you can't compile it until there's an official build, sorry
